Api/appsettings.json

```diff
@@ -6,5 +6,10 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
```

Comment:
We should probably make this a subject to be bound to instances of `ISubmarineAuthenticationSettings`.